### PR TITLE
[MNGSITE-540] Use correct plugin for reporting

### PR DESCRIPTION
**Pull request by kwin**
With https://issues.apache.org/jira/browse/MPLUGIN-423 the reporting mojo has been extracted in a dedicated plugin

### CODE REVIEW

File: maven-scm-plugin/pom.xml [hunk 1]
```diff
@@ -228,7 +228,7 @@
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-plugin-plugin</artifactId>
+        <artifactId>maven-plugin-report-plugin</artifactId>
```
Comment (kwin) [inline maven-scm-plugin/pom.xml:231]: I am not sure why there is no profile `reporting` in this project. IMHO this belongs there. WDYT @michael-o

Comment (slawekjaranowski) [reply maven-scm-plugin/pom.xml:231]: For me is ok ... even more I would be happy to remove reporting profile at all everywhere.

We always should execute all reporting plugins not only in special profile

**Comment by michael-o** [comment]
Will check later today

File: maven-scm-plugin/pom.xml (hunk 1 above)
Comment (michael-o) [reply maven-scm-plugin/pom.xml:231]: @kwin I move this to a profile for consistency reasons with others.

Comment (michael-o) [reply maven-scm-plugin/pom.xml:231]: > For me is ok ... even more I would be happy to remove reporting profile at all everywhere.
> 
> We always should execute all reporting plugins not only in special profile

This is a separate discussion. The reason is likely computational overhead.

**Review by michael-o — CHANGES_REQUESTED** [review]

File: maven-scm-plugin/pom.xml (hunk 1 above)
Comment (kwin) [reply maven-scm-plugin/pom.xml]: I moved to a profile in https://github.com/apache/maven-scm/pull/216/commits/bacee6d36cd28f2a02eedaa726e6f4a21ed75147

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #1249

**Pull request merged**